mteb/tasks/Clustering/multilingual/MLSUMClusteringP2P.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
+
+import datasets
+import numpy as np
+
+from mteb.abstasks import AbsTaskClustering, MultilingualTask, TaskMetadata
+
+_LANGUAGES = {
+    "de": ["deu-Latn"],
+    "fr": ["fra-Latn"],
+    "ru": ["rus-Cyrl"],
+    "es": ["spa-Latn"],
+}
+
```

Comment:
Should we include turkish as well?

> We present MLSUM, the first large-scale MultiLingual SUMmarization dataset. Obtained from online newspapers, it contains 1.5M+ article/summary pairs in five different languages -- namely, French, German, Spanish, Russian, Turkish.

Reply:
I didn't include it because all `topics` values were set to unknown in the dataset, which resulted in an v-measure of 1.0

Reply:
Yep sure!